lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -678,6 +660,25 @@ swift::matchWitness(
     }
   }
 
+  // SWIFT_ENABLE_TENSORFLOW
```

Comment:
Note: I moved the "missing `@differentiable` attribute" check down, so that it also applies to initializer requirements.